congresspeople-app/src/App.test.js

```diff
@@ -1,8 +1,8 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+// import { render, screen } from '@testing-library/react';
+// import App from './App';
```

Comment:
avoid leaving commented code as it is dead code and may lead to confusion in the future

Reply:
I merged that branch with commented lines because ESLINT was causing me conflicts when openning the PR and I did not find a way to solved it, but I'll fix it with the next changes.